proxy/src/main/java/com/wavefront/agent/preprocessor/ReportPointObfuscateTransformer.java

```diff
@@ -60,29 +60,36 @@ public ReportPoint apply(@Nullable ReportPoint reportPoint) {
           String metric = reportPoint.getMetric();
           metric = encode(metric);
           reportPoint.setMetric(metric);
+          ruleMetrics.incrementRuleAppliedCounter();
           break;
+
         case "sourceName":
+        case "hostName":
           String source = reportPoint.getHost();
           source = encode(source);
```

Comment:
Consider using a different var maybe calling it encSource for better readability.